### PR TITLE
chore: roll to 1.15.0-next-1630342926000

### PR DESCRIPTION
**Pull request by kumaraditya303**

### CODE REVIEW

File: playwright/_impl/_api_structures.py [hunk 1]
```diff
@@ -104,6 +104,14 @@ class ResourceTiming(TypedDict):
     responseEnd: float
 
 
+class RequestSizes(TypedDict):
+    requestBodySize: int
```
Comment (mxschmitt) [inline playwright/_impl/_api_structures.py:108]: why sometimes int and sometimes float? it should be all integers.

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools-scm", "wheel==0.36.2", "auditwheel==4.0.0"]
+requires = ["setuptools-scm==6.0.1", "wheel==0.37.0", "auditwheel==4.0.0"]
```
Comment (mxschmitt) [inline pyproject.toml:2]: please exclude that from the PR

File: local-requirements.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
 auditwheel==4.0.0
 autobahn==21.3.1
-black==21.7b0
+black==21.8b0
```
Comment (mxschmitt) [inline local-requirements.txt:3]: please exclude that from the PR

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -10,7 +10,7 @@ repos:
       - id: check-toml
       - id: requirements-txt-fixer
   - repo: https://github.com/psf/black
-    rev: 21.7b0
+    rev: 21.8b0
```
Comment (mxschmitt) [inline .pre-commit-config.yaml:13]: please exclude that from the PR

File: setup.py [hunk 5]
```diff
@@ -163,7 +163,7 @@ def run(self) -> None:
         "write_to": "playwright/_repo_version.py",
         "write_to_template": 'version = "{version}"\n',
     },
-    setup_requires=["setuptools_scm", "wheel"],
+    setup_requires=["setuptools_scm==6.0.1", "wheel==0.37.0"],
```
Comment (mxschmitt) [inline setup.py:166]: please exclude that from the PR.

**Review by mxschmitt — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (kumaraditya303) [reply pyproject.toml:2]: @mxschmitt I had already created a pr to bump versions of which I will rebase so merge that first

File: playwright/_impl/_api_structures.py (hunk 1 above)
Comment (mxschmitt) [reply playwright/_impl/_api_structures.py:108]: fixed in https://github.com/microsoft/playwright/pull/8555

File: pyproject.toml (hunk 2 above)
Comment (mxschmitt) [reply pyproject.toml:2]: Please still revert it. Normally you always base-off from master.

**Comment by kumaraditya303** [comment]
flaky tests:
- `test_should_work_for_subresource`

**Review by mxschmitt — APPROVED** [review]
LGTM, but please add some sync tests (mirror them)

**Pull request merged**